packages/reports/src/Containers/__tests__/open-positions-table.spec.js

```diff
@@ -14,12 +14,6 @@ describe('OpenPositionsTable', () => {
         expect(screen.getByTestId('dt_loading_component')).toBeInTheDocument();
     });
 
-    it('should render "EmptyMessageComponent" when we have no "active_positions"', () => {
-        render(<OpenPositionsTable currency='USD' active_positions={[]} component_icon='IcUnderlying' />);
-        expect(screen.getByText(/you have no open positions yet/i)).toBeVisible();
-        expect(screen.getByTestId('dt_empty_trade_history_icon')).toBeVisible();
-    });
-
```

Comment:
`EmptyMessageComponent` is rendered based on `is_empty` prop and condition to trigger `is_empty = true` is based on `is_empty={active_positions_filtered.length === 0}` and `is_empty={!has_multiplier_contract}`.

Not `active_positions={[]}`. Can leave the test case in as a comment in case It might get used. Since 'open-positions.jsx' will be re-written once we have more contract types.